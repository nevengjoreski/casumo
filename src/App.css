body {
  font-family: 'Tangerine', serif;
  font-size: 12px;
}

.info {
  font-family: 'Bree Serif', serif;
  font-size: 20px;
  width: calc(100vw - 1150px);
  padding: 10px;
  text-align: center;
}

.overlay {
  font-family: 'Tangerine', serif;
  font-size: 90px;
  padding: 40px!important;
  border: 2px solid white!important;
}

.filters button, .filters div {
  margin-left: 20px;
}

.filters {
  text-align: center;
  margin: 10px 0;
  font-family: 'Tangerine', serif;
}

.myButton {
  font-family: 'Bree Serif', serif;
  width: 250px;
  height: 50px;
  box-shadow: 3px 4px 8px -1px #000000;
  background-color: transparent;
  border-radius: 5px;
  border: 1px solid #000000;
  display: inline-block;
  cursor: pointer;
  color: #2b2a2b;
  font-size: 15px;
  padding: 6px 10px;
  text-decoration: none;
  text-shadow: 0px 1px 0px #000000;
}

.myButton:hover {
  background-color: transparent;
}

.myButton:active {
  position: relative;
  top: 1px;
}

.select {
  position: relative;
  display: inline-block;
  margin-bottom: 15px;
  width: 200px;
  height: 50px;
}

/*.shadow{*/
/*    !*box-shadow: 3px 4px 8px -1px #000000;*!*/
/*    box-shadow: 0px -2px 6px 3px rgba(0,0,0,0.75);*/
/*    border: 1px solid #000000;*/
/*}*/
.select select {
  font-family: 'Bree Serif', serif;
  font-size: 15px;
  box-shadow: 3px 4px 8px -1px #000000;
  text-shadow: 0px 1px 0px #000000;
  display: inline-block;
  width: 100%;
  cursor: pointer;
  padding: 10px 10px;
  outline: 0;
  border: 1px solid #000000;
  border-radius: 11px;
  background: #ffffff;
  color: #000000;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
}

.halloween {
  color: black;
  background-color: coral !important;
}

.financeLastFriday {
  color: black;
  background-color: olive !important;
}

.fridayHalloween {
  background: linear-gradient(135deg, olive 20%, coral 80%) !important;
}

.select select::-ms-expand {
  display: none;
}

.select select:hover,
.select select:focus {
  color: #000000;
  background: #ffffff;
}

.select select:disabled {
  opacity: 0.5;
  pointer-events: none;
}

.select_arrow {
  position: absolute;
  top: 16px;
  right: 5px;
  pointer-events: none;
  border-style: solid;
  border-width: 8px 5px 0px 5px;
  border-color: #7b7b7b transparent transparent transparent;
}

.select select:hover ~ .select_arrow,
.select select:focus ~ .select_arrow {
  border-top-color: #000000;
}

.select select:disabled ~ .select_arrow {
  border-top-color: #cccccc;
}