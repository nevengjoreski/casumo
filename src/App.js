import React, {lazy, Suspense} from 'react';
import './App.css';
import Loading from "./components/Loading";

const BigListAgGrid = lazy(() => import('./components/BigListAgGrid'))

function App() {
  return (
      <Suspense fallback={<Loading/>}>
        <BigListAgGrid/>
      </Suspense>

  )
}

export default App;
