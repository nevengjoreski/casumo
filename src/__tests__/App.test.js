import React from "react";
import renderer from "react-test-renderer";
import App from "../App";

describe("App Creation", () => {
  it("should render properly", () => {
    const component = renderer.create(<App />);
    expect(component).toMatchSnapshot();
  });
});
