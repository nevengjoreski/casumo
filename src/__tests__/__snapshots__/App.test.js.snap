// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Creation should render properly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <div
    style={
      Object {
        "fontSize": "100px",
        "paddingTop": "50px",
        "textAlign": "center",
      }
    }
  >
    <p>
      Loading Please Wait
    </p>
    <div
      className="sweet-loading"
    >
      <div
        className="css-0"
      >
        <div
          className="css-oyxt0d"
        />
        <div
          className="css-13c2h83"
        />
        <div
          className="css-1wzt0wo"
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`App should render properly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <div
    style={
      Object {
        "fontSize": "100px",
        "paddingTop": "50px",
        "textAlign": "center",
      }
    }
  >
    <p>
      Loading Please Wait
    </p>
    <div
      className="sweet-loading"
    >
      <div
        className="css-0"
      >
        <div
          className="css-oyxt0d"
        />
        <div
          className="css-13c2h83"
        />
        <div
          className="css-1wzt0wo"
        />
      </div>
    </div>
  </div>
</div>
`;
